src/main/java/org/iac2/common/utility/Edmm.java

```diff
@@ -263,11 +271,18 @@ public static String findHostIp(RootComponent component, DeploymentModel deploym
 
     private static void addPropertyDefinition(Attribute<?> attribute, Entity propertiesEntity) {
         String propertyName = attribute.getName();
-        String edmmType = switch (attribute.getType().getName()) {
-            case "java.lang.Integer", "java.lang.Long" -> DefaultKeys.INTEGER;
-            case "java.lang.Float", "java.lang.Double" -> DefaultKeys.FLOAT;
-            default -> DefaultKeys.STRING;
-        };
+        String edmmType = DefaultKeys.STRING;
+        Class<?> javaType = attribute.getType();
+
+        if (Integer.class.equals(javaType) || Long.class.equals(javaType)) {
```

Comment:
maybe some instanceof would be "more straightforward" but you get a Class<?> anyway

Reply:
I am not sure if `instanceof` is possible in this case... I can only get the type of the attribute and its name but not its value since this describes the attributes of a class not a specific object.